www/templates/self_service_mode_login.html

```diff
@@ -0,0 +1,19 @@
+<ion-view class="company-home" cache-view="false">
+  <subheader></subheader>
```

Comment:
On the phone, the field is underneath the subheader, but hopefully this will be resolved by the other issue?